polaris-react/src/components/Tag/Tag.module.css

```diff
@@ -1,14 +1,10 @@
 .Tag {
-  /* stylelint-disable-next-line -- Polaris component custom properties */
-  --pc-tag-height: 20px;
   position: relative;
   display: inline-flex;
   max-width: 100%;
   align-items: center;
-  /* stylelint-disable-next-line -- Polaris component custom properties */
-  min-height: var(--pc-tag-height);
-  padding: 0;
   padding-inline: calc(var(--p-space-100) + var(--p-space-050));
+  padding-block: var(--p-space-025);
```

Comment:
I think this can be removed.
Double checking Figma, Tag should have 20px height. With the changes in this PR, Tag has 22px height, so removing this should resolve the Chromatic diff.

Reply:
Good check. I tried cleaning this up a bit more to remove the `line-height` hidden in the unnecessary `.Text` class. However, this might cause a visual regression for tags with custom content.

I'll keep unchanged for now and remove the added padding. Thanks, Lo!